.github/workflows/lxd-ci.yml

```diff
@@ -0,0 +1,79 @@
+name: LXD-CI
```

Comment:
Please lint yaml with http://www.yamllint.com/

Reply:
This linter only makes it worse :)
Now yaml file written completely by github actions rules from docs.